src/libFLAC/CMakeLists.txt

```diff
@@ -0,0 +1,132 @@
+cmake_minimum_required(VERSION 3.12)
+
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "[xX]86(_64)?|(AMD|amd)64|i[346]86")
+    option(WITH_AVX "Enable AVX, AVX2 optimizations" ON)
+    option(WITH_SSE "Enable AVX, AVX2 optimizations" ON)
+endif()
+
+check_c_compiler_flag(-msse2 HAVE_MSSE2_FLAG)
+if(HAVE_MSSE2_FLAG AND WITH_SSE)
+    add_compile_options(-msse2)
```

Comment:
What about MSVC?

Reply:
Do you know a similar flag for cl compiler?

Reply:
You mean [/arch:SSE2](https://docs.microsoft.com/en-us/cpp/build/reference/arch-x86?view=vs-2019) ?

Reply:
Thanks. Added.